common/configuration-gui.adoc

```diff
@@ -0,0 +1,19 @@
+The options under the *Configuration* menu allow you to configure global options for your {product-title} environment, view diagnostic information, and view analytics on the servers in the environment. The menu displays the {product-title} environment at the enterprise, zone, and server levels.
+
+There are four main areas:
+
+* *Settings*
++
+This menu enables you to modify the configuration of your {product-title} infrastructure. You can also create analysis profiles and schedules for these profiles.
++
+* *Access Control*
++
+This menu enables you to modify the configuration of your {product-title} infrastructure. You can also create analysis profiles and schedules for these profiles.
++
+* *Diagnostics*
++
+This menu displays the status of your servers and their roles and provides access to logs.
++
+* *Database*
++
+This menu enables you to modify the configuration of your {product-title} infrastructure. You can also create analysis profiles and schedules for these profiles.
```

Comment:
No they shouldn't. But I'm just single-sourcing that content from the General Configuration doc:

https://github.com/ManageIQ/manageiq_docs/blob/master/doc-General_Configuration/topics/Configuration.adoc

Thanks for pointing this out. Now that it's single-sourced, we can fix it simultaneously in both documents.